src/asleep/get_sleep.py

```diff
@@ -69,8 +69,8 @@ def get_parsed_data(raw_data_path, info_data_path, resample_hz, args):
         data = data.reset_index()
 
         # apply time shift
-        start_time = pd.to_datetime(info['StartTime'])
-        end_time = pd.to_datetime(info['EndTime'])
+        start_time = pd.to_datetime(data['time'].iloc[0])
+        end_time = pd.to_datetime(data['time'].iloc[-1])
```

Comment:
This always overwrites the start and end time, even for the axivity device. Perhaps this should only be done for csv files.

Reply:
This is PR still active? 

Also, to avoid using magic numbers, we can name 0, and -1 as `start_time_idx` or `end_time_idx`.